README.md

```diff
@@ -684,7 +684,7 @@ jobs:
 
 [![start example](https://github.com/cypress-io/github-action/workflows/example-start/badge.svg?branch=master)](.github/workflows/example-start.yml)
 
-**Note:** GitHub cleans up the running server processes automatically. This action does not stop them.
+**Note:** A server continues to run until the end of the GitHub workflow job that started it. At the end of the job the GitHub workflow runner executes a "Complete job" phase automatically where it terminates any server processes which are still running.
```

Comment:
maybe also provide tips for:

- how to stop a server started by `cypress-io/github-action`:
  - using multiple jobs
  - using a command to allow stopping between steps in a single job

otherwise I think this paragraph will be too easily missed and it will still be a weird behavior

in my opinion, even an open, unresolved issue is better than this hidden paragraph of unusual, unexpected behavior

Reply:
@karlhorky 

Thanks for your feedback. I think the topic of manually stopping a server started through `github-action` needs to be handled separately. This PR is really just intended to document correctly how the action currently works.

If there is a use-case which requires killing a server then that should be documented with an example to make it clear how to do it. That would however overload this PR.

Reply:
I think the problem is that this PR is marked as closing https://github.com/cypress-io/github-action/issues/360, which was the open issue that users were finding when encountering this unusual behavior.

Maybe @nagash77 or @chrisbreiding can reopen #360.

It is not resolved and this confusing behavior remains, without a documented way of addressing it (as I mentioned in my comment above).